app.py

```diff
@@ -16,15 +20,44 @@
   "required": ["firstname"]
 }
 
+schema2 = {
+  "type": "object",
+  "properties": {
+    "path": { "type": "string" }
+  },
+  "required": ["path"]
+}
+
+#homepage
 @app.route('/')
 def hello_world():
     return 'Hello, World!'
     
+#take json input (according to schema) and return json
 @app.route("/json", methods=['POST'])
 @expects_json(schema)
 def read_json():
     timestamp = datetime.datetime.now().isoformat()
     if 'lastname' in request.json:
         return jsonify({"greeting": "Hello, " + request.json['firstname']+" "+request.json['lastname'], "timestamp":timestamp})
     else:
-        return jsonify({"greeting": "Hello, " + request.json['firstname'], "timestamp":timestamp})
\ No newline at end of file
+        return jsonify({"greeting": "Hello, " + request.json['firstname'], "timestamp":timestamp})
+    #curl -H "Content-Type: application/json" --data '{"firstname": "Marie","lastname":"Gronemeyer","age":29}' http://127.0.0.1:5000/json
+
+#run some jobs in a rq
+@app.route("/rq")
+def try_rq():
+  q = Queue(connection=Redis())
+  result = q.enqueue(add, 5,5)
+  result2 = q.enqueue(count_lines, 'poppunk/input_data/qfile.txt')
+  return "All jobs submitted\n"
```

Comment:
Can you retrieve the output from the submitted jobs?

Might be worth trying to change this to
* Return an ID from the submitted job (I am assuming result is the ID of a task on the queue?)
* Add another endpoint which takes that ID and returns the status of the task, I think rq should have a function to get you the status
* Add a third endpoint which takes the ID and returns the result of the run - you might have to change your `add` and `count_lines` functions to return a value instead of printing

For this to work your Queue object might need to initialised once when the app starts up and re-used by different endpoints.